### DISCUSSION TITLE
indentation error when org-indent turned on

### DISCUSSION THREAD

**Discussion by titaniumbones** [minad/org-modern#160] [Q&A] [answered]
I reckon I may be missing something, but it looks like indentation is off by one character per level when org-modern-mode and org-indent-mode are both active. Is this expected? With org-modern-hide-stars set to "do not hide stars" the issue is slightly different, with off-by-one errors in the opposite direction starting at level 3. 
![image](https://github.com/minad/org-modern/assets/364340/2dfac840-74ee-403d-8580-a731e55f3ad6)

![image](https://github.com/minad/org-modern/assets/364340/24ece267-094f-43c5-aaa0-f68459be1227)

**Comment by minad** [comment, marked as answer]
Yes, I've observed this before. The problem is that the prettified stars do not have the right size in your font, such that the text becomes misaligned. Maybe the indentation depth can be somehow reconfigured in org-indent-mode? I am not using org-indent-mode often, so I cannot tell if this is possible. Maybe @jdtsmith has an idea, since he uses org-indent-mode and even made [org-modern-indent](https://github.com/jdtsmith/org-modern-indent)?

**Comment by titaniumbones** [comment]
Thanks for responding so quickly.  I just found some of the closed issues about this and I can see it's complicated.

setting `org-indent-indentation-per-level` gets it pretty close to exactly right.  Turning on org-modern-indent-mode works pretty well too.  It doesn't seem like there's an exact solution but this is pretty good, thanks.  Marking as answered.

**Comment by jdtsmith** [comment]
`org-indent-mode` simply adds the right number of spaces as a line-prefix, e.g. for depth 1 headings, `#("*" 0 1 (face org-indent))` (where org-indent face is simply hidden).  In principle you could "change the size" of this prefix to help with alignment, but before that I'd change the font, or select a new fancy character with the same fixed width as `*` and SPC.  You can discover the width of a character by typing it in and `(posn-object-width-height (posn-at-point))`.
